From 2302c9d522c714f9f8d6b8c28f528637cb62360b Mon Sep 17 00:00:00 2001
Message-Id: <patch-1.1-2302c9d522-20210607T143157Z-avarab@gmail.com>
In-Reply-To: <20210420233830.2181153-1-bga@google.com>
References: <20210420233830.2181153-1-bga@google.com>
From: =?UTF-8?q?=C3=86var=20Arnfj=C3=B6r=C3=B0=20Bjarmason?=
 <avarab@gmail.com>
Date: Mon, 7 Jun 2021 16:28:26 +0200
Subject: [PATCH] protocol-caps.h: add newline at end of file
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Add a trailing newline to the protocol-caps.h file added in the recent
a2ba162cda (object-info: support for retrieving object info,
2021-04-20). Various editors add this implicitly, and some compilers
warn about the lack of a \n here.

Signed-off-by: Ævar Arnfjörð Bjarmason <avarab@gmail.com>
---
 protocol-caps.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/protocol-caps.h b/protocol-caps.h
index 6351648e37..0a9f49df11 100644
--- a/protocol-caps.h
+++ b/protocol-caps.h
@@ -7,4 +7,4 @@ struct packet_reader;
 int cap_object_info(struct repository *r, struct strvec *keys,
 		    struct packet_reader *request);
 
-#endif /* PROTOCOL_CAPS_H */
\ No newline at end of file
+#endif /* PROTOCOL_CAPS_H */
-- 
2.32.0.rc3.434.gd8aed1f08a7

